package.json

```diff
@@ -27,6 +27,7 @@
     "eslint-plugin-jsx-a11y": "1.2.2",
     "eslint-plugin-promise": "1.3.1",
     "eslint-plugin-react": "5.2.2",
+       "eslint-plugin-react-intl": "1.0.2",
```

Comment:
this indentation looks a little funny. An extra 2 spaces, maybe? Do you mind updating